src/App.js

```diff
@@ -1,25 +1,68 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from 'react';
+
+const api = {
+  key: "ed01eea6a7f49ba69a36b2ee1e8c889b",
+  base: "https://api.openweathermap.org/data/2.5/"
+}
 
 function App() {
+  const [query, setQuery] = useState('');
+  const [weather, setWeather] = useState({});
+
+  const search = e => {
+    if (e.key === "Enter") {
+      fetch(`${api.base}weather?q=${query}&units=metric&appid=${api.key}`)
+        .then(res => res.json())
+        .then(result => {
+          setWeather(result);
+          setQuery('');
+          console.log(result);
+        });
+    }
+  }
+
+  const dateBuilder = (d) => {
+    let months = ["January", "February", "March", "April", "May", "June", "July",
+      "August", "September", "October", "November", "December"];
+    let days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+
+    let day = days[d.getDay()];
+    let date = d.getDate();
+    let month = months[d.getMonth()];
+    let year = d.getFullYear();
+
+    return `${day} ${date} ${month} ${year}`
+  }
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <main>
+        <div className='Search-box'>
+          <input
+            type='text'
+            className='search-bar'
+            placeholder='search...'
+            onChange={e => setQuery(e.target.value)}
+            value={query}
+            onKeyDown={search}
+          />
+        </div>
+        {(typeof weather.main !== "undefined") ? (
+          <div>
+            <div className='Date-box'>
+              <div className='location'>{weather.name}, {weather.sys.country}</div>
+              <div className='date'>{dateBuilder(new Date())}</div>
+            </div>
+            <div className='Weather-box'>
+              <div className='temp'>{Math.round(weather.main.temp)}°C</div>
+              <div className='weather'>{weather.weather[0].main}</div>
+            </div>
+          </div>
+        ) : ('')}
```

Comment:
```suggestion
        {weather.main !== undefined && (
          <div>
            <div className='Date-box'>
              <div className='location'>{weather.name}, {weather.sys.country}</div>
              <div className='date'>{dateBuilder(new Date())}</div>
            </div>
            <div className='Weather-box'>
              <div className='temp'>{Math.round(weather.main.temp)}°C</div>
              <div className='weather'>{weather.weather[0].main}</div>
            </div>
          </div>
        )}
```

굳이 삼항연산자를 사용하지 않아도 될 것 같아요!